FlatBuffers.cmake

```diff
@@ -0,0 +1,11 @@
+set(FlatBuffers_Include ${PROJECT_SOURCE_DIR}/third_party/flatbuffers/include)
```

Comment:
nitpick, can we put cmake related things inside a folder? something like `./cmake`